src/measure_theory/integral/set_to_l1.lean

```diff
@@ -857,11 +1296,159 @@ begin
   exact L1.set_to_L1_indicator_const_Lp hT hs hμs x,
 end
 
+lemma set_to_fun_const [is_finite_measure μ] (hT : dominated_fin_meas_additive μ T C) (x : E) :
+  set_to_fun μ T hT (λ _, x) = T univ x :=
+begin
+  have : (λ (_ : α) , x) = set.indicator univ (λ _, x), from (indicator_univ _).symm,
+  rw this,
+  exact set_to_fun_indicator_const hT measurable_set.univ (measure_ne_top _ _) x,
+end
+
 @[continuity]
 lemma continuous_set_to_fun (hT : dominated_fin_meas_additive μ T C) :
   continuous (λ (f : α →₁[μ] E), set_to_fun μ T hT f) :=
 by { simp_rw L1.set_to_fun_eq_set_to_L1 hT, exact continuous_linear_map.continuous _, }
 
+/-- Auxiliary lemma for `set_to_fun_congr_measure` -/
```

Comment:
Can you still comment in the docstring on what the lemma says? It is pretty hard to parse :-)